crates/nao/src/lib.rs

```diff
@@ -324,6 +324,7 @@ impl Nao {
     pub async fn flash_image(&self, image_path: impl AsRef<Path>) -> Result<()> {
         let status = self
             .rsync_with_nao(false)
+            .arg("--copy-links")
```

Comment:
Does this only allow you to give a symlink as image input file or will it also affect what will be copied to the NAO? We probably don't want to copy symlinks to the NAO because the Device Firmware Upgrade is not trustworthy :wink:

Reply:
From the `rsync` manual page:
> --copy-links, -L         transform symlink into referent file/dir

And it does seem to work that way. I tested it by building an new image and flashing it that way.

Reply:
Ok, thanks! Could you please test what happens in the destination directory? We should be sure to not copy a symlink to the directory.

Reply:
Don't have a nao to test right now, but when testing locally, it indeed turns the link into the file its pointing to when copying. There are no extraneous symlinks at the destination.

Reply:
Great, fine for me. Thanks for testing. Resolving.